wp-web-push/wp-web-push-main.php

```diff
@@ -5,8 +5,16 @@
 
 class WebPush_Main {
   private static $instance;
+  public static $ALLOWED_TRIGGERS;
 
   public function __construct() {
+    self::$ALLOWED_TRIGGERS = array(
+      array('text' => __('New Post'), 'key' => 'new-post', 'enable_by_default' => true, 'hook' => 'publish_post', 'action' => 'send_new_post'),
+      array('text' => __('New Comment'), 'key' => 'new-comment', 'enable_by_default' => false, 'hook' => 'wp_insert_comment', 'action' => 'send_new_comment'), // To Do:  Is this useless without saying which post?  Surely not all posts...
+      array('text' => __('Updated Post'), 'key' => 'update-post', 'enable_by_default' => false, 'hook' => 'save_post', 'action' => 'send_updated_post')
```

Comment:
What happens if you select both `New Post` and `Updated Post`? Are both the `publish_post` and `save_post` hooks going to be triggered when you publish a post?
If that's the case, we need to take that into account and only send the notification once.